coalib/bears/requirements/PackageRequirement.py

```diff
@@ -25,6 +34,12 @@ def check(self):
         """
         Check if the requirement is satisfied.
 
+        >>> PR = PackageRequirement('pip', 'coala_decorators', '0.2.1')
+        >>> PR.check()
+        Traceback (most recent call last):
+        ...
+        NotImplementedError
+
```

Comment:
add both doctests to the according commit please that adds the class and function :+1: